tests/conftest.py

```diff
@@ -32,3 +33,11 @@ def _strip_invisible(string: str) -> str:
         return string.strip().replace(zero_width_space, "")
 
     return _strip_invisible
+
+
+WindowsPyPathPattern = re.compile(r"[a-z_\\]+\.py")
+
+
+def fix_paths(line: str) -> str:
+    """On Windows the path separator is \\, not /. This makes all python paths in mypy output use / for consistency."""
+    return WindowsPyPathPattern.sub(lambda l: l.group().replace("\\", "/"), line)
```

Comment:
Also just noticed, the variable name isn't accurate. The function is not operating on singular lines, but on the whole of stdout/stderr. Could we rename to `text`?